docs/general/profiles.md

```diff
@@ -81,6 +81,7 @@ They may be used in any Profiles record they apply to.
 | [`pprof.mapping.has_functions`](/docs/registry/attributes/pprof.md) | boolean | Indicates that there are functions related to this mapping. |  | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
 | [`pprof.mapping.has_inline_frames`](/docs/registry/attributes/pprof.md) | boolean | Indicates that there are inline frames related to this mapping. |  | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
 | [`pprof.mapping.has_line_numbers`](/docs/registry/attributes/pprof.md) | boolean | Indicates that there are line numbers related to this mapping. |  | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
+| [`pprof.profile.comment`](/docs/registry/attributes/pprof.md) | string[] | Free-form text associated with the profile. This field should not be used to store any machine-readable information, it is only for human-friendly content. | `["hello world", "bazinga"]` | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
```

Comment:
Do we want to explicitly mention that in terms of `AnyValue` the value should be an array of strings?

Reply:
As the `Type` of the new attribute `pprof.profile.comment` is set to `string[]` I think we don't need to care about `AnyValue` here. 
Looking at `browser.brands` as an example, it will always be handled as array of strings: https://github.com/open-telemetry/opentelemetry-go/blob/84e3f3ac8b25204f3a0f77a805437a5e08573b35/semconv/v1.37.0/attribute_group.go#L1883-L1888

Reply:
Ah, great - thanks!